packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SingleImageViewer/SingleImageViewer.js

```diff
@@ -62,6 +64,8 @@ function SingleImageViewer (props) {
                 scale={scale}
                 height={height}
                 width={width}
+                subject={subject}
+                viewBox={viewBox}
```

Comment:
Maybe add `viewBox` to the `SVGContext.Provider` so that it’s available to all the consumers of `SVGContext`?

Reply:
Then `LineControls` can consume the context in order to get the current view box and transforms.

Reply:
The context provider is here, by the way. Storing `viewBox` in the context object might be more convenient than passing it down as a prop. It'll be automatically made available to any child component that imports and uses the `SVGContext` context.
https://github.com/zooniverse/front-end-monorepo/blob/26d66b5b510db74a70dea1d73e4f5c2c1cf2bbec/packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/SingleImageViewer/SingleImageViewer.js#L35

Reply:
Good suggestion. Incorporated